frame/lending/src/currency.rs

```diff
@@ -11,12 +11,16 @@
 /// type BCOIN = Currency<54321, 12>;
 ///
 /// let one_hundred_a_tokens = ACOIN::units(100);
-/// let one_value_of_b = BCOIN::one();
+/// let one_value_of_b = BCOIN::ONE;
 /// ```
 #[derive(Debug, Clone, Copy)]
 pub struct Currency<const ID: u128, const EXPONENT: u8> {}
 
 impl<const ID: u128, const EXPONENT: u8> Currency<ID, EXPONENT> {
+	/// The id of the currency. This is fairly arbitrary, and is only used to differentiate between
```

Comment:
is it considered to be moved later to runtime to replace currency? may be add comment about it

Reply:
The name `RuntimeCurrency` was the best I could come up with for an equivalent of `Currency` with runtime-determined values, it has nothing to do with the runtimes. I see the confusion though, any name suggestions are welcome